CalculatorTestCode/ViewController.swift

```diff
@@ -7,13 +7,85 @@
 
 import UIKit
 
-class ViewController: UIViewController {
 
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        // Do any additional setup after loading the view.
-    }
+class ViewController: UIViewController {
 
+        override func viewDidLoad() {
+            super.viewDidLoad()
+            // Do any additional setup after loading the view.
+        }
 
+        @IBOutlet weak var firstNumberTextField: UITextField!
+        
+        @IBOutlet weak var secondNumberTextField: UITextField!
+        
+        @IBOutlet weak var resultLabel: UILabel!
+        
+        @IBAction func addButton(_ sender: Any) {
+            let firstNumberInput: Int? = Int(firstNumberTextField.text!)
+            let secondNumberInput: Int? = Int(secondNumberTextField.text!)
+            let operation = firstNumberInput! + secondNumberInput!
+                
+            let firstNumberInt = firstNumberInput!
+            let secondNumberInt = secondNumberInput!
+                
+                resultLabel.text = "\(firstNumberInt) + \(secondNumberInt) = \(operation)"
+            }
+            
+            @IBAction func substractButton(_ sender: Any) {
+                let firstNumberInput: Int? = Int(firstNumberTextField.text!)
+                let secondNumberInput: Int? = Int(secondNumberTextField.text!)
+                
+                let firstNumberInt = firstNumberInput!
+                let secondNumberInt = secondNumberInput!
+                
+                let operation = firstNumberInput! - secondNumberInput!
+                    
+                resultLabel.text = "\(firstNumberInt) - \(secondNumberInt) = \(operation)"
+            }
+            
+            @IBAction func multiplyButton(_ sender: Any) {
```

Comment:
```suggestion
            @IBAction func multiplyButtonTapped(_ sender: Any) {
```
 명사보단 동사나 형용사 같은 함수명이 좋아보여요~ 함수 자체가 곱셈 버튼이 눌렸을때 일어나는 일을 의미하기때문에요~~